Modules/posixmodule.c

```diff
@@ -5231,19 +5425,34 @@ os.posix_spawn
     file_actions: object = None
         A sequence of file action tuples.
     /
-
+    *
+    setpgroup: object = None
+        The pgroup to use with the POSIX_SPAWN_SETPGROUP flag.
+    resetids: object = False
+        If the value is `True` the POSIX_SPAWN_RESETIDS will be activated.
```

Comment:
Also POSIX_SPAWN_RESETIDS etc will not say much to Python developers. It would be better to use a very shorten version of the documentation. E.g.: "If True reset effective user and group IDs".